### PR TITLE
Updated extensions native-messaging content

### PR DESCRIPTION
**Pull request by jm-trd-ms**
Updated formatting for extensions native-messaging content.

### CODE REVIEW

File: microsoft-edge/extensions-chromium/developer-guide/native-messaging.md [hunk 1]
```diff
@@ -38,13 +38,11 @@ Add the nativeMessaging permission to the extension’s **manifest.json** file.
...
-```
+```  
 
-### Step 2: Create your native messaging host manifest file
+## Step 2 - Create your native messaging host manifest file  
     
-Native applications must provide a native messaging host manifest file. This manifest file contains the path to the native messaging host executable, the method of communication with the extension, and a list of allowed extensions it can communicate with. Browsers read and validate the native messaging host manifest, but it’s never installed or managed by the browser. 
-The host manifest file must be a valid json file containing the following fields.
-
+Native applications must provide a native messaging host manifest file.  the manifest file contains the path to the native messaging host runtime, the method of communication with the extension, and a list of allowed extensions to which it communicates.  The browser reads and validates the native messaging host manifest.  The browser does not install or manage the manifest file.
```
Comment (gebansal) [inline microsoft-edge/extensions-chromium/developer-guide/native-messaging.md:45]: The?

Comment (gebansal) [inline microsoft-edge/extensions-chromium/developer-guide/native-messaging.md:45]: Native applications must provide a native messaging host manifest file.  The manifest file contains the path to the native messaging host runtime, the method of communication with the extension, and a list of allowed extensions to which it communicates.  The browser reads and validates the native messaging host manifest.  The browser does not install or manage the native messaging host manifest file.

File: microsoft-edge/extensions-chromium/developer-guide/native-messaging.md [hunk 2]
```diff
@@ -58,79 +56,165 @@ The host manifest file must be a valid json file containing the following fields
...
+      Specifies the list of extensions that have access to the native messaging host.  To enable your application to identify and communicate with an extension, in your native messaging host manifest file set the following value.  
+      
+      ```json
+      "allowed_origins": ["chrome-extension://{microsoft_catalog_extension_id}"]
+      ```  
+   :::column-end:::
+:::row-end:::  
+
+Sideload your extension to test native messaging with the host.  
+To sideload of your extension during development, complete the following steps.
```
Comment (gebansal) [inline microsoft-edge/extensions-chromium/developer-guide/native-messaging.md:116]: To sideload of your extension => To sideload your extension

Comment (gebansal) [reply microsoft-edge/extensions-chromium/developer-guide/native-messaging.md:116]: To sideload your extension during development and retrieve microsoft_catalog_extension_id, complete the following steps.

**Review by gebansal — APPROVED** [review]

File: microsoft-edge/extensions-chromium/developer-guide/native-messaging.md (hunk 1 above)
Comment (jm-trd-ms) [reply microsoft-edge/extensions-chromium/developer-guide/native-messaging.md:45]: Updated.

```md
Native applications must provide a native messaging host manifest file.  The manifest file contains the path to the native messaging host runtime, the method of communication with the extension, and a list of allowed extensions to which it communicates.  The browser reads and validates the native messaging host manifest.  The browser does not install or manage the native messaging host manifest file.  
```

File: microsoft-edge/extensions-chromium/developer-guide/native-messaging.md (hunk 2 above)
Comment (jm-trd-ms) [reply microsoft-edge/extensions-chromium/developer-guide/native-messaging.md:116]: Updated.

```md
Sideload your extension to test native messaging with the host.  
To sideload your extension during development and retrieve `microsoft_catalog_extension_id`, complete the following steps.  
```

**Comment by jm-trd-ms** [comment]
#sign-off offline and merged.

**Pull request merged**